### ISSUE TITLE
 How to use Tesseract_planning to plan a mobile manipulator with 9 degrees of freedom？

### ISSUE THREAD

**Issue by Jm20010201** [tesseract-robotics/tesseract_planning#484]
I tried to modify the example basic_cartesian_example, but I encountered the following problem
![2024-07-25 17-18-08 的屏幕截图](https://github.com/user-attachments/assets/81037369-73f8-4f82-9790-85e23dc2add2)
Do you know how to set it up? I tried to use the ompl library but it takes about 12 seconds to plan. Is there any good way? Thank you very much！！！

**Comment by Levi-Armstrong** [comment]
Yes it is very much possible, but it will require some setup. Is the intent to do point-to-point or are you wanting to use all degrees of freedom to have the tool center point follow a cartesian path?

**Comment by Levi-Armstrong** [comment]
It can plan for just the base, just the manipulator or base and manipulator. This is configured by how you define the manipulators in the srdf. Then you use this name to indicate what manipulator group you trying to plan with. I would recommend starting with the freespace example and provide final joint pose then progress from there. If you plan to use a cartesian point as the goal you will need to define a more efficient IK sovler for your system. You should be able to use the robot on a positioner IK solver for this application.
